decidim-core/app/commands/decidim/create_report.rb

```diff
@@ -72,11 +72,6 @@ def hideable?
       hidden_by_admin? || (!@reportable.hidden? && moderation.report_count >= Decidim.max_reports_before_hiding)
     end
 
-    def hide!
-      @tool.hide!
-      @tool.send_notification_to_author
```

Comment:
Authors should receive an email when their content is hidden. As far as I see this is still happening: 

> (Bug!) See that you don't have the emails for the Proposal and the Comments moderated to the authors of those. 

(From #14520)

Reply:
I have checked, and the notification was there, but was caught by the notification system. 
I tested with a proposal: 
![image](https://github.com/user-attachments/assets/1d8c8884-2e0f-49fc-b9a4-5b3bd8d479e4)

![image](https://github.com/user-attachments/assets/c6738ffa-b5ab-4ec7-ade8-af02ae8a3681)


I have tested a comment with the following author specs.

- notifications_sending_frequency="daily"
- notifications_sending_frequency="none"
- notifications_sending_frequency="real_time"

I have tested a proposal, and the notification was sent. 

@andreslucena could you give it another spin ?

Reply:
Oh I see it now! I think it is sent with like 10 seconds of delay and to be honest I just saw the first emails and didn't catch the last one ("Your comment has been removed"). You can ignore this part, thanks for your help!